src/context/LandingPageAnimationContext.js

```diff
@@ -0,0 +1,47 @@
+import React, { createContext, useState } from 'react';
```

Comment:
Wouldn't it to be a bit more cleaner if we would handle state like that:
```
...

const [isAnimating, setIsAnimating] = useState(false);

const startAnimation = () => setIsAnimating(true);
const stopAnimation = () => setIsAnimating(false);

return <Provider value={{isAnimating, animateAndRedirect, stopAnimation}} .../>
```

Reply:
Fixed in: 77ef0ba7824fa0a176a840bf5eede7bdc1e52151